cmd/api/src/api/v2/auth/oidc_test.go

```diff
@@ -116,3 +121,57 @@ func TestManagementResource_CreateOIDCProvider(t *testing.T) {
 			ResponseStatusCode(http.StatusInternalServerError)
 	})
 }
+
+func TestManagementResource_DeleteOIDCProvider(t *testing.T) {
+	var (
+		url               = "/api/v2/sso/providers/oidc/%s"
+		mockCtrl          = gomock.NewController(t)
+		resources, mockDB = apitest.NewAuthManagementResource(mockCtrl)
+	)
+
+	t.Run("successfully delete an OIDCProvider", func(t *testing.T) {
```

Comment:
Can we include some users tied to an OIDC provider and also verify that the FK properly deletes them?